### PR TITLE
chore: Dev-Target for creating sample CLI options asset

### PR DESCRIPTION
**Pull request by bitsandfoxes**
#skip-changelog

### CODE REVIEW

File: Directory.Build.targets [hunk 1]
```diff
@@ -489,6 +489,15 @@ void PrintFailedTests(XElement element)
     </Task>
   </UsingTask>
 
+  <!-- Checks if the environment variable 'SENTRY_AUTH_TOKEN' has been set and creates the SentryCliOptions.asset for the sample project
+       This is meant for developwers - so that they don't have to configure the cli options after each clean checkout (or git clean).
```
Comment (vaind) [inline Directory.Build.targets:493]:
```suggestion
       This is meant for developers - so they don't have to configure the CLI options after each clean checkout (or git clean).
```

File: scripts/create-sentry-cli-options.ps1 [hunk 2]
```diff
@@ -0,0 +1,58 @@
+Set-StrictMode -Version latest
```
Comment (vaind) [inline scripts/create-sentry-cli-options.ps1:1]:
```suggestion
Set-StrictMode -Version latest
$ErrorActionPreference = "Stop"
```

File: scripts/create-sentry-cli-options.ps1 [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+$guid = ([Regex]::Match($meta_content, '(?<=guid: )\S+'))
+
+if($guid.Success)
+{
+  Write-Output "Successfully read GUID '$guid' from '$meta_path'"    
+}
+else 
+{
+  Write-Error "Failed to retrieve the guid from '$meta_path'"
+  return
```
Comment (vaind) [inline scripts/create-sentry-cli-options.ps1:16]:
```suggestion
  Write-Error "Failed to retrieve the guid from '$meta_path'"
```

**Review by vaind — APPROVED** [review]

File: scripts/create-sentry-cli-options.ps1 (hunk 2 above)
Comment (bitsandfoxes) [reply scripts/create-sentry-cli-options.ps1:1]: That's neat. Thanks!

**Pull request merged**